tests/conftest.py

```diff
@@ -104,6 +104,15 @@ def remote_addr():
     return "1.2.3.4"
 
 
+@pytest.fixture
+def remote_addr_hashed():
+    """
+    Static output of `hashlib.sha256(remote_addr.encode("utf8")).hexdigest()`
+    Created statically to prevent needing to calculate it every run.
+    """
+    return "6694f83c9f476da31f5df6bcc520034e7e57d421d247b9d34f49edbfc84a764c"
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏it's not that much of a time-suck, might be able to run the real thing?

```shellsession
$ python -m timeit -c 'import hashlib;hashlib.sha256("1.2.3.4".encode("utf8")).hexdigest()'
500000 loops, best of 5: 758 nsec per loop
```

Reply:
I decided against it, since there's a test that manually sets `remote_addr` to `None` for some reason. I'll look into that is and when the test is re-examined for hashed values.